### ISSUE TITLE
Game is stuck on loading screen.

### ISSUE THREAD

**Issue by aleaallee** [sailro/EscapeFromTarkov-Trainer#32]
After downloading the compiled packages and changing the file name for the live version of the game and copying the files in the folder the readme indicated, I started the game and it won't go past the first loading screen.

**Comment by sailro** [comment]
- Could you give your exact EFT/sptarkov versions ?
- Regarding the trainer, which release did you download ?
- Could you summarize what you copied exactly and where ?
- Do you see any error in `%LOCALAPPDATA%Low\Battlestate Games\EscapeFromTarkov\output_log.txt` ?

**Comment by aleaallee** [comment]
Hi, I downloaded the latest release, my EFT version is 0.12.10.2.212646.
I dropped the files in my EFT folder, and renamed NLog.dll.nlog-live to NLog.dll.nlog.
Regarding the errors, the errors I see are mostly type/token errors.

**Comment by sailro** [comment]
Do not use this trainer with the live version, you'll be banned. Install sptarkov v1.4.4, and you should be good.
